applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -406,8 +408,12 @@ class ExplicitCentralDifferencesScheme
         mTime.Current = r_current_process_info[TIME];
         mTime.Delta = r_current_process_info[DELTA_TIME];
 
-        mTime.Middle = 0.5 * (mTime.Previous + mTime.Current);
+        mTime.Middle   = mTime.Current - 0.50*mTime.Delta;   //0.5 * (mTime.Previous + mTime.Current);
```

Comment:
```suggestion
        mTime.Middle   = mTime.Current - 0.50*mTime.Delta;
```

please don't leave commented code in master, this is very confusing for someone else trying to understand whats going on